### PR TITLE
users -> people

### PR DESCRIPTION
**Pull request by torgo**
this is an experimental PR to demonstrate what it might look like to change "users" to "people" across thd document. 

Referencing #52

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/69.html" title="Last updated on Nov 3, 2021, 2:49 PM UTC (5b75327)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/69/9cc8254...5b75327.html" title="Last updated on Nov 3, 2021, 2:49 PM UTC (5b75327)">Diff</a>

### CODE REVIEW

**Review by wseltzer — APPROVED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -581,26 +582,26 @@
...
-their users need. For example, some user agents may help their users present different
+people need. For example, some user agents may help people present different
 [=identities=] to subdivisions of a single [=site=].
 
 <div class="issue" data-number="1">
 
 There is disagreement about whether [=user agents=] may also widen their [=machine-enforceable
 contexts=]. For example, some user agents might want to help their users present a single
-[=identity=] to multiple [=sites=] that the user understands represent a single [=party=], or to a
+[=identity=] to multiple [=sites=] that people understand represent a single [=party=], or to a
```
[the remark this answers is not in the document]
Comment (darobin) [reply index.html:598]: I still think that `user` is better in this context.

File: index.html [hunk 2]
```diff
@@ -1114,13 +1115,13 @@
...
 
-if the user doesn't intend for it to do so.
+if the person using it doesn't intend for it to do so.
 
 ## Powerful capabilities {#hl-capabilities}
 
 Contributes to [=misattribution=].
 
-For example, a site that sends SMS without the user's intent could cause them to
+For example, a site that sends SMS without someone's intent could cause them to
```
[the remark this answers is not in the document]
Comment (darobin) [reply index.html:1126]: Same. I still think that `user` is better in this context.

[the remark this answers is not in the document]
Comment (darobin) [reply index.html:1126]:
```suggestion
For example, a site that sends SMS without the user's intent could cause them to
```

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (darobin) [reply index.html:598]:
```suggestion
[=identity=] to multiple [=sites=] that the user understands represent a single [=party=], or to a
```

**Review by darobin — CHANGES_REQUESTED** [review]
I think the two remaining comments are cases where keeping `user` would be a lot better. After that, it's all good!

**Review by darobin — APPROVED** [review]

**Pull request merged**